assets/l10n/intl_en.arb

```diff
@@ -3724,7 +3724,7 @@
     },
     "noTeachersFound": "No teachers found to report to",
     "pleaseEnterANumber": "Please enter a number greater than 0",
-    "archiveRoomDescription": "The chat will be moved to the archive. Other users will be able to see that you have left the chat.",
```

Comment:
Small thing - when changing pre-existing copy in this file, you should also update the pre-existing copy in intl_es.arb (the spanish localizations)